### ISSUE TITLE
Cannot repreduce the result of fashion by training from scratch

### ISSUE THREAD

**Issue by Jarvisss** [RenYurui/Global-Flow-Local-Attention#68]
Hi, anyone has trained the model from scratch?

Followed IMAGE_GENERATION.md, I download the same dataset of fashion, and train the model from scratch,

I first pretrain the flow generator for ~500k iters and then train the whole network for 100K iterations,
![image](https://user-images.githubusercontent.com/16323003/100217688-b379f180-2f4e-11eb-9ed2-4ffed8dfd9cb.png)

![image](https://user-images.githubusercontent.com/16323003/100211315-f1731780-2f46-11eb-839a-5ded328d41e9.png)

But by using the pretrained model, I can get result:
![image](https://user-images.githubusercontent.com/16323003/100217307-48301f80-2f4e-11eb-9683-e9fe8ff1e8fe.png)

Could anyone please tell me how many iterations should I train?

**Comment by RenYurui** [comment]
Hi Jarvisss,
It seems that the difference is caused by the flow generator. 
Does the sampling correctness loss coverage after ~500k iters?
I think training the flow generator for a longer time may help with the results.
Thanks.

**Comment by Jarvisss** [comment]
Maybe I should train another 500k iters, Thanks for your advice, 
I wonder what is the sampling correctness loss when it converges in your experiment ant the value?

**Comment by Jarvisss** [comment]
Another interesting thing is, if I use bilinear sampling in the sample correctness loss, the loss even cannot converge
